README.md

```diff
@@ -5,7 +5,7 @@ Write down two truths and one lie below.
 
 Once you're done, submit a pull request (do NOT commit directly to the main branch)! We'll try to guess the lie before we merge your branch with the main branch.
 
-YOUR NAME HERE:
-1. A truth (or a lie)
-2. A lie (or a truth)
-3. Another truth (or your lie)
+Christofer Brothers:
+1) Swimming is my 3rd favorite sport
+2) I fractured my wrist on a trampoline
+3) Monk is my favorite DnD Class
```

Comment:
2) False. You fractured your wrist while punching a guy in a full suit of armor while yelling "stunning strike!"

Reply:
I suspect #3